docker-compose.yml

```diff
@@ -98,16 +98,16 @@ services:
   restarters_db:
     profiles: ["core", "debug", "discourse"]
     image: mysql:5.7.33
-    container_name: ${MYSQL_CONTAINER_NAME}
```

Comment:
Careful with these. docker-compose can act a bit weird with templating and escaping.

Reply:
Addressed in [2913b13](https://github.com/iFixit/restarters/pull/25/commits/2913b1315c958dd7611be9b608ab60ad8cc15504)